crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -19,7 +89,7 @@ impl Wallet {
 
         let rows: Vec<models::Bid> = bids::table
             .select(bids::all_columns)
-            .filter(bids::bidder_address.eq(&self.address))
+            .filter(bids::bidder_address.eq::<String>(self.address.clone().into()))
```

Comment:
Needed the into and ::<String> compiler was does not implement diesel::expression::Expression<T>